README.md

```diff
@@ -25,16 +25,18 @@ which contains a docker compose setup for running a charon cluster on your local
 If however, you want to build from source with this repo directly, you can get started with:
 
 ```bash
-# Install go 1.18 or later (on mac with homebrew installed)
+# Install go 1.18 or later (https://go.dev/doc/install)
+# If on mac, you can also install using homebrew
 brew install go
 
 # Build the charon binary
 go build -o charon
 
-# Use charon's create-cluster command to generate a local simnet cluster.
+# Use charon's create cluster command to generate a local charon cluster
+./charon create cluster
```

Comment:
not sure this is very helpful, what is the dev going to do with a cluster_lock file? Suggesting removing this section and just pointing them to charon-docker-compose

Reply:
so, remove the whole "build directly from source" section?